portal.go

```diff
@@ -1824,3 +1867,21 @@ func (portal *Portal) HandleNewDisappearingMessageTime(newTimer uint32) {
 		intent.SendNotice(portal.MXID, fmt.Sprintf("Disappearing messages set to %s", exfmt.Duration(time.Duration(newTimer)*time.Second)))
 	}
 }
+
+func (portal *Portal) HasRelaybot() bool {
+	return portal.bridge.Config.Bridge.Relay.Enabled && len(portal.RelayUserID) > 0
+}
+
+func (portal *Portal) addRelaybotFormat(userID id.UserID, content *event.MessageEventContent) bool {
+	member := portal.MainIntent().Member(portal.MXID, userID)
+	if member == nil {
+		member = &event.MemberEventContent{}
+	}
+	content.EnsureHasHTML()
```

Comment:
To fix captions after rebasing on main

```suggestion
	if content.FileName == "" {
		content.FileName = content.Body
	}
	content.EnsureHasHTML()
```